Assets/SEE/XR/RadialSelection.cs

```diff
@@ -0,0 +1,409 @@
+using SEE.Controls.Actions;
+using SEE.GO;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.InputSystem;
+using UnityEngine.UI;
+using UnityEngine.XR.Interaction.Toolkit;
+using SEE.Utils;
+
+/// <summary>
+/// This class is used, to open a
+/// radial menu in VR.
+/// This script is based on this tutorial: https://www.youtube.com/watch?v=n-xPN1v3dvA
+/// </summary>
+public class RadialSelection : MonoBehaviour
+{
+    /// <summary>
+    /// The number of radial-parts.
+    /// </summary>
+    int numberOfRadialPart = 0;
```

Comment:
Missing visibility modifier. Also, the variable should probably be called `numberOfRadialParts`.